docs/DeveloperGuide.md

```diff
@@ -7,32 +7,97 @@
+|v2.1|Expert user|Update ratings of travel packages based on customer feedback|Provide a better recommendations for future customers|
+|v2.1|Expert user|Ascertain the error in my input based on the error messages from the application|Quickly troubleshoot any mistakes that would slow down my work|
+|v2.1|Expert user|View all reservations currently under a specific travel package|Optimize that travel package according to user’s feedback|
+|v2.1|Expert user|I can build a customized travel package based on what the customer wants|It appeals to customers who do not want our existing tour packages|
+|v2.1|Expert user|Add in hotels to the list of available hotel options|Update the itinerary |
+|v2.1|Expert user|Add in countries to the list of available countries|Update the itinerary|
+|v2.1|Expert user|Sort all reservations according to month|Analyze trends throughout the year|
+|v2.1|Expert user|Mark a reservation as ‘completed’|Access both completed and ongoing reservations|
+|v2.1|Expert user|Compute the price of custom travel packages|Get a rough estimate for what to charge the customer|
 
 ## Non-Functional Requirements
 
-{Give non-functional requirements}
+### Performance Requirements
+- Should work on any mainstream OS as long as it has Java 11 or above installed.
+
+### Usability
+- A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+- Prompts and instructions must be in proper and concise sentences for ease of understanding.
 
 ## Glossary
 
 * *glossary item* - Definition
+* Packages - Travel Package within the agency's database
+* Reservations - Reservation of travel package made by one customer through the app 
 
 ## Instructions for manual testing
 
 {Give instructions on how to do a manual product testing e.g., how to load sample data to be used for testing}
+
+### Summary of User Commands
+
+| Command | Format Examples |
+| ------- | --------------- |
+|packages|packages|
+|info|info {num} (num < number of available packages) <br/> e.g. info 2 |
+|add|add {package_name} {country} {duration} {price} {vacancies} <br/>  e.g. add Skiing_Trip Sweden 15/2/2022-19/2/2022 800 100|
+|delete|delete {num} (num < number of available packages) <br/> e.g. delete 2|
+|reserve|reserve {package_number} {contact_name} {contact_number} {number_pax} <br/> e..g reserve 3 John 91234567 3 |
+|remove|remove {reservation_id} <br/> e..g remove R001|
+|reservations|reservations {package_number} <br/> eg. reservations 2|
+
+## Feature - Help Command 
+Aim: Displays a list of all available commands that the user can refer to as a guide
+
+## Feature - Storage
+#### Initialisation (Loading Data)
+The sequence diagram that shows how `Storage` is created and the data is loaded from the saved files when the program is initialised is shown below: </br>
+![](StorageSeqDiag.png)
```

Comment:
- Since this is an instance of the `Storage` class, it should be written as :Storage
- There should be a constructor shown that would instantiate an object of the `Storage` class
- The call to the constructor should have a return line that would return the object
- The call to the constructor should be represented by a solid line

<img width="223" alt="Screenshot 2022-03-31 at 11 27 46 AM" src="https://user-images.githubusercontent.com/49554923/160970492-5afd664b-cab9-4573-961e-1086f9910a45.png">